### DISCUSSION TITLE
Enhancements to QaraTMS

### DISCUSSION THREAD

**Discussion by libregeek** [a13xh7/QaraTMS#21] [Ideas]
I did an initial round of testing and evaluation of QaraTMS and the application looks impressive though there isn't a GA release. I would really like to see Qara as a stable Open Source application for test case management. Following are the features that I would like to suggest:

1. Add a field for storing actual results of testing. This could be an optional Text field that tester can use when they update the status of test execution. A new table may be required to store the actual results.
2. Assign a test suite execution to a user. This is per plan and it is also an optional field. The current schema may have to be modified to accommodate this change and there could also be UI changes to the Test Plan edit screen.
3. Send the failed test cases to a bug tracker (possibly to a mailbox and let the bug tracker pick it from there
4. DB Normalization.
5. Test Run Report with all the details of execution.

These are the current items that are in my mind. If you think that these make sense then I can also contribute to the project.

**Comment by a13xh7** [comment]
Good ideas but I don't have time for this project right now, maybe later.

**Comment by libregeek** [comment]
If you are okay then I can submit PR for the above enhancements.

**Comment by a13xh7** [comment]
That would be great
